src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -16,14 +20,23 @@
 public class ArmSubsystem extends SubsystemBase {
   private CANSparkMax armMotor;
   private Solenoid brake;
-  private DutyCycleEncoder encoder;  
-  
+  private DutyCycleEncoder encoder;
+  private PIDController pid;
+
 
+ 
+  private static final int kCanID = Constants.ARM_MOTOR;
+  private static final MotorType kMotorType = MotorType.kBrushless;
```

Comment:
These two variables are not needed as we directly use them below instead, they can be deleted.